devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -277,6 +277,11 @@ bool WholeBodyDynamicsDevice::openEstimator(os::Searchable& config)
 
     yInfo() << "wholeBodyDynamics : Loading model from " << modelFileFullPath;
 
+    // Add fixed joints l_foot_ft_sensor, r_foot_ft_sensor in case they are omitted because the corresponding F/T sensors are not included in the model
+    std::vector<std::string> additionalConsideredJoints{"l_foot_ft_sensor", "r_foot_ft_sensor"};
+    // Append additionalConsideredJoints to estimationJointNames
+    estimationJointNames.insert(estimationJointNames.end(), additionalConsideredJoints.begin(), additionalConsideredJoints.end());
```

Comment:
I understand this fix, but I believe this fix is very specific to iCub based URDF models. So maybe we can pass these `additionalConsideredJoints` itself as an optional parameter in the configuration file?

Reply:
@traversaro what's your opinion on this?

Reply:
Just noticed https://github.com/robotology/whole-body-estimators/pull/109#issuecomment-818592941